prog/weavewait/main.go

```diff
@@ -26,6 +26,11 @@ func main() {
 		args = args[1:]
 	}
 
+	if len(args) > 0 && args[0] == "-c" { // tell pid 1 to continue
+		checkErr(syscall.Kill(1, syscall.SIGUSR2))
```

Comment:
I believe we could get the pid from `Container.State.Pid` when we launch the exec.